src/api/app/views/layouts/webui/_status_message_announcement.html.haml

```diff
@@ -0,0 +1,12 @@
+- if current_announcement
+  .row.justify-content-center#status-message-announcement
+    .col-12
+      .alert.fade.show.alert-notice
+        .row
+          .col-md-10
+            %i.fa.fa-bullhorn.text-info
+            = truncate(current_announcement.message, length: 80)
```

Comment:
Why are you truncating the message?

Reply:
I was afraid it was to long for a flash. But it's true the user won't be able to read the whole text before accepting it (if he/she doesn't realize it is completely displayed in the News box). @hennevogel what do you think?

Reply:
You worry too much :smile: